src/Controller/SearchEngineController.php

```diff
@@ -1,33 +1,96 @@
 <?php
 
-// src/Controller/DefaultController.php
 namespace App\Controller;
 
 // Import TNTSearch
 use TeamTNT\TNTSearch\TNTSearch;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use App\Entity\Usergroup;
 
+use App\Service\SearchEngineManager;
+use Symfony\Component\HttpFoundation\Request;
+
+
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 
 class SearchEngineController extends AbstractController
-{    
+{
+    private $formFactory;
+
+    /**
+	 * @Route(
+     * "/search/{searchQuery}",
+     * name="search_page",
+     * defaults={"searchQuery" = ""},
+     * requirements={"searchQuery"=".+"}
+     * )
+     *
+ 	 * @param \Symfony\Component\HttpFoundation\Request $request
+   	 * @param \App\Service\SearchEngineManager      $searchEngineManager
+	 *
+	 * @return \Symfony\Component\HttpFoundation\Response
+	 */
+	public function searchPage (
+        Request $request,
+        SearchEngineManager $searchEngineManager,
+        string $searchQuery
+	) {
+        $searchQuery = $request->request->get('searchQuery');
+
+        // TODO: Add Pagination to results
+		$page     = $request->query->get( 'page', 0 );
+		$per_page = 20;
+        $filters = $request->query->get( 'form', [] );
+
+        unset( $filters[ 'submit' ] );
+
+        // If request comes from header search bar
+        if ($searchQuery != '') {
+            $filters[ 'keywords' ] = explode( ',',  $searchQuery  );
+        }
+        // If request is done from search page
+        else if ( !empty( $filters[ 'query' ] ) ) {
```

Comment:
```suggestion
        elseif ( !empty( $filters[ 'query' ] ) ) {
```
En php `else if` (un `if` dans un `else`) est un poil différent de `elseif`

Reply:
je savais pas, je fais les changements aux endroits où j'ai fait la confusion